File: Algoritmos Traduzidos/Perl/Algoritmos_Luis/61.pl
#Entrar com a razão de urna PG e o valor do 1º termo. Calcular e imprimir o 5° ter¬mo da série.

print "introduza o 1o termo\n";
$termo = <STDIN>;
print "introduza a razao\n";
$razao = <STDIN>;
$quinto = $termo * ($razao**4);
print "O 5o termo desta progressao geometrica e: $quinto\n";

File: Algoritmos Traduzidos/Ruby/Algoritmos_Luis/61.rb
#Entrar com a razão de urna PG e o valor do 1º termo. Calcular e imprimir o 5° termo da série.
puts "introduza o 1o termo"
termo = gets
puts "introduza a razao"
razao = gets
quinto = termo.to_f * (razao.to_f**4)
puts "O 5o termo desta progressao geometrica e: #{quinto}"